server_tests/test/clientTests.js

```diff
@@ -308,7 +317,7 @@ describe("RUST API TEST SUITE", function() {
     });
 
     // CLOSE SESSION
-    describe("Testing /osc/commands/execute camera.closeSession endpoint", function() {
+    describe("Testing /osc/commands/execute camera.closeSession endpoint. Disabled for testing over Wifi", function() {
```

Comment:
The update test should indeed fail, and is a test that should only be executed for mock cameras, alongside shutdown.